ban/tests/http/test_postcode.py

```diff
@@ -47,3 +47,46 @@ def test_postcode_select_use_default_orderby(get, url):
     assert resp.json['collection'][0]['code'] == '90101'
     assert resp.json['collection'][1]['municipality']['insee'] == "90002"
     assert resp.json['collection'][2]['municipality']['insee'] == "90001"
+
+
+@authorize
+def test_get_postcode_collection_filtered_by_1_code_param(get, url):
+    PostCodeFactory(code='90000')
+    PostCodeFactory(code='91000')
+    param = dict(code='90000')
```

Comment:
If you have only one param, you can just pass it directly to query_string, without the need for a variable:

    ..., query_string={'code': 'xxx'})

Also, better to use a plural for a multivalue, so `params` would be.